spring-cloud-netflix-core/src/main/java/org/springframework/cloud/netflix/zuul/filters/RouteOptions.java

```diff
@@ -0,0 +1,16 @@
+package org.springframework.cloud.netflix.zuul.filters;
+
+import java.util.LinkedHashMap;
+import java.util.LinkedHashSet;
+import java.util.Map;
+import java.util.Set;
+
+import lombok.Data;
+
+@Data
+public class RouteOptions {
+
+	private Set<String> allowedMethods = new LinkedHashSet<>();
```

Comment:
Having anything hardcoded, precludes a user from adding their own. Again, see my previous link to spring-cloud-gateway.